packages/grafana-ui/src/components/Logs/LogRow.tsx

```diff
@@ -59,57 +71,65 @@ class UnThemedLogRow extends PureComponent<Props, State> {
   ) {
     const {
       getRows,
+      onClickFilterLabel,
+      onClickFilterOutLabel,
       highlighterExpressions,
-      onClickLabel,
       row,
       showDuplicates,
-      showLabels,
       timeZone,
       showTime,
       theme,
     } = this.props;
-    const { showContext } = this.state;
+    const { showDetails, showContext } = this.state;
     const style = getLogRowStyles(theme, row.logLevel);
     const showUtc = timeZone === 'utc';
 
     return (
-      <div className={cx([style.logsRow])}>
+      <div className={style.logsRow}>
         {showDuplicates && (
-          <div className={cx([style.logsRowDuplicates])}>
+          <div className={style.logsRowDuplicates}>
             {row.duplicates && row.duplicates > 0 ? `${row.duplicates + 1}x` : null}
           </div>
         )}
-        <div className={cx([style.logsRowLevel])} />
-        {showTime && showUtc && (
-          <div className={cx([style.logsRowLocalTime])} title={`Local: ${row.timeLocal} (${row.timeFromNow})`}>
-            {row.timeUtc}
-          </div>
-        )}
-        {showTime && !showUtc && (
-          <div className={cx([style.logsRowLocalTime])} title={`${row.timeUtc} (${row.timeFromNow})`}>
-            {row.timeLocal}
+        <div className={style.logsRowLevel} />
+        {
+          <div onClick={this.toggleDetails} className={style.logsRowToggleDetails}>
+            <i className={showDetails ? 'fa fa-chevron-up' : 'fa fa-chevron-down'} />
           </div>
-        )}
-        {showLabels && (
-          <div className={cx([style.logsRowLabels])}>
-            <LogLabels
+        }
+        <div>
```

Comment:
Outer div is not needed.

Reply:
Actually it is needed, cause it creates css table-cell that includes 2 rows -> row with time and message and row with logs details.